tests/test_virtualenv.py

```diff
@@ -2,11 +2,19 @@
 import shutil
 import tempfile
 from flexmock import flexmock
-from pyp2rpm.virtualenv import (DirsContent, VirtualEnv, site_packages_filter,
-                                scripts_filter)
+try:
+    from pyp2rpm.virtualenv import (DirsContent,
+                                    VirtualEnv,
+                                    site_packages_filter,
+                                    scripts_filter)
+except ImportError:
+    VirtualEnv = None
 from pyp2rpm.name_convertor import NameConvertor
 from pyp2rpm.settings import DEFAULT_DISTRO, DEFAULT_PYTHON_VERSION
 
+pytestmark = pytest.mark.skipif(VirtualEnv is None,
```

Comment:
Shouldn't you actually use the marker? Or how does it work, what tests are skipped?

Reply:
The whole module is skipped as documented here https://docs.pytest.org/en/latest/skipping.html#skip-all-test-functions-of-a-class-or-module.